spec/class.dd

```diff
@@ -439,27 +425,30 @@ $(GNAME Constructor):
     $(OL
         $(LI Storage is allocated for the object.
         If this fails, rather than return $(D null), an
-        $(D OutOfMemoryError) is thrown.
+        $(LINK2 ../library/core/exception/out_of_memory_error.html, OutOfMemoryError) is thrown.
```

Comment:
@wilzbach do we have a canonical way for such links?

Reply:
Yup - `$(REF OutOfMemoryError, core, exception)` or `$(REF_ALTTEXT OutOfMemoryError, OutOfMemoryError, core, exception)`  (though that will link to Ddoc, not Ddox which the link currently uses)

Reply:
Aha, build failed with:

Error while processing file: /dev/shm/dtest/work/repo/dlang.org/web/dlangspec.html
object.Exception@tools/chmgen.d(588): Attempting to escape site root (dereferencing ../library/core/exception/out_of_memory_error.html from dlangspec.html)

Reply:
so... `$(ROOT_DIR)`?